presto-main/src/main/resources/webapp/assets/loader.css

```diff
@@ -0,0 +1,64 @@
+/**
+The MIT License (MIT)
+
+Copyright (c) 2014 Luke Haas
```

Comment:
Should this file be in the vendor directory? I assume it came from somewhere else